honeybee_radiance/sensorgrid.py

```diff
@@ -31,23 +32,20 @@ class SensorGrid(object):
         * directions
         * room_identifier
         * light_path
+        * mesh
     """
 
     __slots__ = ('_identifier', '_display_name', '_sensors', '_room_identifier',
-                 '_light_path')
+                 '_light_path', '_mesh')
 
     def __init__(self, identifier, sensors):
         """Initialize a SensorGrid."""
         self.identifier = typing.valid_rad_string(identifier)
         self._display_name = None
-        self._sensors = tuple(sensors)
-        for sen in self._sensors:
-            if not isinstance(sen, Sensor):
-                raise ValueError(
-                    'Sensors in SensorGrid must be form type Sensor not %s' % type(sen)
-                )
+        self.sensors = sensors
         self._room_identifier = None
         self._light_path = None
+        self._mesh = None
```

Comment:
I feel we might want to add a `classmethod` to create a sensor grid from an input mesh. That will come handy in several cases.

Reply:
Good point. Will do.

Reply:
Done